src/backend/move_logic.js

```diff
@@ -11,23 +11,19 @@ function Move(type, player, to) {
 }
 
 let registeredTurn = -1;
-export function registerMoveCallback(roomName, turn, playerID) {
-	console.log(`register move callback: ${turn} ${registeredTurn}`);
+// TODO: get actual number of players
+let numPlayers = 2;
```

Comment:
@preetshah123 if you can get the numPlayers into the function (or someone else) that would be great